app/config/assets/css/odk-survey.css

```diff
@@ -629,3 +629,28 @@ Eventually we should make a jqm theme for this.
     overflow:hidden;
     margin-right:50px;
 }
+
+.savepoints.finalized {
+    display: flex;
+    text-align: center;
+    align-items: center;
+    width: fit-content;
+    padding: 16px;
+    justify-content: center;
+    color: #1CCB3D;
+    background-color: #F0FDF2;
+    height: 40px;
+}
+
+.savepoints.incomplete {
+    display: flex;
+    text-align: center;
+    align-items: center;
+    width: fit-content;
+    padding: 16px;
+    justify-content: center;
+    color: #DF961F;
+    background-color: #FCF8EA;
+    height: 40px;
+}
```

Comment:
@juayuohcarineneng19 I don't see any border-radius, did you leave this out intentionally?

Those states have a border-radius on the screen but I can't see it here

Reply:
> @juayuohcarineneng19 I don't see any border-radius, did you leave this out intentionally?
> 
> Those states have a border-radius on the design screen but I can't see it here

Hi @Redeem-Grimm-Satoshi  I have added border-radius .